src/Utility/ExprFunctions.C

```diff
@@ -141,6 +141,59 @@ std::vector<std::string> splitComps (const std::string& functions,
 }
 
 
+/*!
+  \brief Helper template to get size and dimension of a return type.
+*/
+
```

Comment:
Insistent spacing between comment and first code line (one blank line her, none in the following)